build-logic/convention-plugins/src/main/kotlin/social.androiddev.library.gradle.kts

```diff
@@ -7,26 +10,12 @@ val libs = the<org.gradle.accessors.dm.LibrariesForLibs>()
 kotlin {
     jvm("desktop")
     android()
-    iosX64()
-    iosArm64()
-    iosSimulatorArm64()
+    ios()
```

Comment:
does ios target accumulate all others (x64, arm, simulators)? If so, that's awesome

Reply:
yep!